lib/modules/idea-overview-widgets/index.js

```diff
@@ -1,23 +1,5 @@
 const googleMapsApiKey = process.env.GOOGLE_MAPS_API_KEY;
 
-const merge            = require('lodash.merge');
-
-const getPosterImageUrl = (idea, googleMapsApiKey) => {
-      return idea.imageUrl ? idea.imageUrl : idea.location && idea.location && idea.location.coordinates ? 'https://maps.googleapis.com/maps/api/streetview?'+
-                           'size=800x600&'+
-                           `location=${idea.location.coordinates[0]},${idea.location.coordinates[1]}&`+
-                           'heading=151.78&pitch=-0.76&key=' + googleMapsApiKey
-                         : null;
-
-}
-
-const enrichIdeas = (ideas, googleMapsApiKey) => {
-  return ideas.map((idea) => {
-    idea.posterImageUrl = getPosterImageUrl(idea, googleMapsApiKey)
-    return idea;
-  });
-};
-

```

Comment:
Why did you remove these lines?

Reply:
This is unused code, I already removed it once, but because of [this](https://github.com/Amsterdam/openstad-frontend/commit/cd2e6de29a531f7f746125231c977644a0513d99) merge it was back again :-)